frontend/lib/identification/identification_page.dart

```diff
@@ -72,4 +100,15 @@ class IdentificationPage extends StatelessWidget {
       mode: LaunchMode.externalApplication,
     );
   }
+
+  Future<void> _openRemoveCardDialog(BuildContext context) async {
+    final userCodeModel = Provider.of<UserCodeModel>(context, listen: false);
+    await RemoveCardConfirmationDialog.show(
+        context: context, userCode: userCodeModel.userCodes[cardIndex], carouselController: carouselController);
+  }
+
+  void _moveCarouselToLastPosition() {
+    final userCodeModel = Provider.of<UserCodeModel>(context, listen: false);
+    carouselController.jumpToPage(userCodeModel.userCodes.length);
```

Comment:
Indices not starting with zero? :astonished:

Reply:
ah okay yes i should add a >0 check so i can avoid the other issue above :)